easybuild/toolchains/linalg/intelmkl.py

```diff
@@ -84,6 +85,7 @@ def _set_blas_variables(self):
         interfacemap = {
             TC_CONSTANT_INTELCOMP: 'intel',
             TC_CONSTANT_GCC: 'gf',
+            TC_CONSTANT_PGI: 'intel',
```

Comment:
please include `https://www.pgroup.com/support/link.htm#mkl` in a comment above this line?

it looks wrong at first sight, but you're right, it's correct